pypy_patch.py

```diff
@@ -18,7 +18,7 @@
 
 regex = re.compile(r"(?P<name>pp3[py0-9_]+-pypy[3_p0-9]+)")
 
-for file in pathlib.Path("./wheels").iterdir():
+for file in pathlib.Path("./dist").iterdir():
```

Comment:
This hack can be removed after https://github.com/PyO3/maturin/commit/6099945a281f45943172d295718cb2ee52bdddcb released.

Reply:
super, I'd only mentioned the situation, not the issue/commit. Thanks for that!